src/ripple/net/impl/RPCCall.cpp

```diff
@@ -1265,8 +1314,15 @@ rpcClient(std::vector<std::string> const& args,
             if (!setup.client.admin_password.empty ())
                 jvRequest["admin_password"] = setup.client.admin_password;
 
-            jvParams.append (jvRequest);
+            if (!jvRequest.isMember("ripplerpc"))
```

Comment:
I'm a little confused as to what these lines are doing.  Is the idea that for v2 requests (have `"ripplerpc"` field), then the request parameters are found in the `jvRequest["params"]` field but in the v1 call the entire `jvRequest` object is the parameter?